src/mp2_grids.F

```diff
@@ -254,7 +254,7 @@ SUBROUTINE get_minimax_grid(para_env, unit_nr, homo, Eigenval, num_time_points,
             ALLOCATE (weights_cos_tf_t_to_w(num_freq_points, num_time_points))
             weights_cos_tf_t_to_w = 0.0_dp
 
-            num_points_per_magnitude = mp2_env%ri_rpa_im_time%num_points_per_magnitude
+            num_points_per_magnitude = 200
```

Comment:
It is supposed to be a parameter? If yes, it is common to use the following line in the declaration block to prevent the variable from accidental changes:
`INTEGER, PARAMETER :: num_points_per_magnitude = 200`

Reply:
Done, see below